### PR TITLE
convert /profile from BML to TT

### PR DESCRIPTION
**Pull request by kareila**
Apart from a few structural glitches that I fixed, this is intended to be visually identical to the current version of the page.

### CODE REVIEW

**Comment by rahaeli** [comment]
Yay!!!!! (But also, "this is intended to be visually identical to the current version of the page" oh god lol)

**Comment by kareila** [comment]
It’s just another step of the journey.

**Comment by rahaeli** [comment]
the real reward was the 500 comments of people calling us morons we made along the way

File: views/profile/main.tt [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+  [% PROCESS content_block opts = content_block_opts %]
+[%- END -%]
+
+[%- # MAINTAINERS/MODERATORS (COMMUNITY ONLY) -%]
+
+[%- maint_userids = profile.maintainer_userids;
+    mod_userids   = profile.moderator_userids;
+    admin_userids = maint_userids.merge( mod_userids );
+
+    IF admin_userids.size;
```
Comment (zorkian) [inline views/profile/main.tt:264]: I am in awe at your TT wielding. This is intense.

**Pull request merged**